### PR TITLE
Update current-federal-non-dotgov.csv

### PR DESCRIPTION
**Pull request by cfx47**
PT has requested that we add the following domains so that they show up in their Trustymail and HTTPS reports. I verified they are not currently being picked up in the sources gathered.

goldengateclub.com
presidio-foods.com
presidio-residences.com
presidioofficersclub.com
presidioofsanfrancisco.com
presidioofsanfrancisco.net
presidioofsanfrancisco.org
presidioresidences.com
presidiotours.com
presidiotours.org
presidiotrust.net
presidiotrust.org
presidiotrust.tours
presidiotunneltops.com
presidiotunneltops.org
presidioworkspaces.com
thepresidioofsanfrancisco.com
thepresidioofsanfrancisco.net
thepresidioofsanfrancisco.org
presidiolodging.com
presidiogolf.com
presidiobus.com

#  #

## 🗣 Description ##
Add domains to Presidio Trust scope so that they show up in their Trustymail and HTTPS reports.

## 💭 Motivation and context ##
Requested by Presidio Trust to ensure they are scanning these non .gov domains.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

### CODE REVIEW

File: current-federal-non-dotgov.csv [hunk 1]
```diff
@@ -473,6 +473,7 @@ globalprioritymail.us,Federal Agency - Executive,United States Postal Service,,W
 globalpriority.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 globalunitedstatesmail.com,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 globalunitedstatesmail.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+goldengateclub.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
```
Comment (dav3r) [inline current-federal-non-dotgov.csv]: Remove extraneous space:
```suggestion
goldengateclub.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
```

File: current-federal-non-dotgov.csv [hunk 2]
```diff
@@ -1154,6 +1155,24 @@ powertodobusiness.org,Federal Agency - Executive,United States Postal Service,,W
...
+presidioofsanfrancisco.org,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidioresidences.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotours.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotours.org,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotrust.net,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotrust.org,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotrust.tours,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotunneltops.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidiotunneltops.org,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+presidioworkspaces.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
```
Comment (dav3r) [inline current-federal-non-dotgov.csv]: Remove extraneous spaces:
```suggestion
presidio-foods.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidio-residences.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiobus.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiogolf.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiolodging.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioofficersclub.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioofsanfrancisco.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioofsanfrancisco.net,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioofsanfrancisco.org,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioresidences.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotours.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotours.org,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotrust.net,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotrust.org,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotrust.tours,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotunneltops.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidiotunneltops.org,Federal Agency - Executive,Presidio Trust,,Washington,DC,
presidioworkspaces.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
```

File: current-federal-non-dotgov.csv [hunk 3]
```diff
@@ -1358,6 +1377,9 @@ thepowertodobusiness.com,Federal Agency - Executive,United States Postal Service
 thepowertodobusiness.net,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 thepowertodobusiness.org,Federal Agency - Executive,United States Postal Service,,Washington,DC,
 thepowertodobusiness.us,Federal Agency - Executive,United States Postal Service,,Washington,DC,
+thepresidioofsanfrancisco.com,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+thepresidioofsanfrancisco.net,Federal Agency - Executive, Presidio Trust,,Washington,DC,
+thepresidioofsanfrancisco.org,Federal Agency - Executive, Presidio Trust,,Washington,DC,
```
Comment (dav3r) [inline current-federal-non-dotgov.csv]: Remove extraneous spaces:
```suggestion
thepresidioofsanfrancisco.com,Federal Agency - Executive,Presidio Trust,,Washington,DC,
thepresidioofsanfrancisco.net,Federal Agency - Executive,Presidio Trust,,Washington,DC,
thepresidioofsanfrancisco.org,Federal Agency - Executive,Presidio Trust,,Washington,DC,
```

**Review by dav3r — CHANGES_REQUESTED** [review]
You've got some extra spaces that need to be removed.

**Comment by dav3r** [comment]
@cfx47 Please also update the PR description so that all relevant checkboxes are checked and all non-relevant ones are deleted.  Thanks!

File: current-federal-non-dotgov.csv (hunk 2 above)
Comment (dav3r) [reply current-federal-non-dotgov.csv]: This suggestion was missed - please include it.

**Review by dav3r — APPROVED** [review]
LGTM

I have a couple of notes for the future.  When you commit suggestions, please always replace the default commit message ("Update current-federal-non-dotgov.csv", which is not very helpful) with a more useful message (e.g. "Remove extra spaces").

Also, we **strongly** encourage everyone to follow [these commit message rules](https://cbea.ms/git-commit/#seven-rules) which specify the use of the imperative mood (i.e. you should write "Remove extra spaces" rather than "Remove**d** extra spaces").

**Review by jsf9k — APPROVED** [review]

**Pull request merged**